File: Intermediate Concepts/array3.c
#include <stdio.h>

#define MEALS 3

int main()
{
    int accumulator = 0; 
    puts("Calorie Counter");

    for(int i = 0; i < MEALS; i++)
    {
        int calorie;
        printf("Calories at meal %d: ",i+1);
        scanf("%d", &calorie);
        accumulator += calorie;
    }
    printf("You had a total of %d calories.\n", accumulator);
    

    return(0);
}

File: Intermediate Concepts/array3.js
// Counter calories 
const prompt = require('./prompt').prompt

async function counterCalories() {

    let accumulator = 0 
    for(let i = 0; i < 3; i++){
        let calorie = await prompt(`Calories at meal ${i}: `)        
        accumulator += parseInt(calorie)
    }
    console.log(`You had a total of ${accumulator} calories.`)
    readline.close()
}

counterCalories()